digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficLightService.scala

```diff
@@ -128,6 +114,16 @@ class TrafficLightService(val roadLinkService: RoadLinkService) extends PointAss
     }
   }
 
+  def getProperty(properties: Seq[Property], property: String) : Option[PropertyValue] = {
+    properties.find(p => p.publicId == property).get.values.map(_.asInstanceOf[PropertyValue]).headOption
+  }
```

Comment:
Doesn't exist yet a generic getProperty?
I think we already have done a method to be used each time we need to extract a property by public Id!

Reply:
Modified the one on PointAssetOperations to be able to use that one instead of creating a new one